include/swift/AST/DiagnosticsParse.def

```diff
@@ -1466,11 +1466,11 @@ ERROR(attr_implements_expected_member_name,PointsToFirstBadToken,
 ERROR(attr_differentiable_expected_function_name,PointsToFirstBadToken,
       "expected a qualified %0 function", (StringRef))
 ERROR(attr_differentiable_expected_parameter_list,PointsToFirstBadToken,
-      "expected a list of parameters to differentiate with respect to, e.g. (.0, w, b)", ())
+      "expected a list of parameters to differentiate with respect to", ())
 ERROR(attr_differentiable_use_wrt_not_withrespectto,none,
       "use 'wrt:' to specify parameters to differentiate with respect to", ())
 ERROR(attr_differentiable_expected_parameter,PointsToFirstBadToken,
-      "expected a parameter, which can be the index of a function parameter with a leading dot (e.g. '.0'), or 'self'", ())
+      "expected a parameter, which can be a function parameter name or 'self'", ())
 ERROR(attr_differentiable_parameter_index_must_be_positive_integer,PointsToFirstBadToken,
       "parameter index must be a positive integer", ())
 ERROR(attr_differentiable_nondifferentiable_function,none,
```

Comment:
This and the next diagnostics seem to be obsolete and have no users. Can you help remove?

Reply:
Thanks! Done in https://github.com/apple/swift/pull/22000/commits/aec0072e4eb43e7c59c6b73d980030413deaceea.